ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeViewshed/AnalyzeViewshed.html

```diff
@@ -0,0 +1,5 @@
+<html>
+	<img src="AnalyzeViewshed.png"></img>
+	<hr>
+	<p>This sample demonstrates how to use GeoprocessingTask to calculate a viewshed using a geoprocessing service. Click any point on the map to see all areas that are visible within a 1 kilometer radius. It may take a few seconds for the model to run and send back the results.</p>
```

Comment:
it's referring to the GP model that is behind the GP service, but I agree it is confusing, especially with Qt's heavy usage of the word model. I'll change it.